firmware-binaries/test-cases/registerwbc_test/build.rs

```diff
@@ -0,0 +1,37 @@
+// SPDX-FileCopyrightText: 2022 Google LLC
+//
+// SPDX-License-Identifier: Apache-2.0
+
+use std::env;
+use std::fs;
+use std::path::{Path, PathBuf};
+
+use memmap_generate::memory_x_from_memmap;
+
+fn memmap_dir() -> PathBuf {
+    PathBuf::from(env!("CARGO_MANIFEST_DIR"))
+        .join("../../../")
+        .join("_build")
+        .join("memory_maps")
+}
+
+/// Put the linker script somewhere the linker can find it.
+fn main() {
+    let memory_x = memory_x_from_memmap(
+        memmap_dir().join("VexRiscv.json"),
```

Comment:
Shouldn't this point to the memory map from the test?

Reply:
Absolutely!